include/h2o/socket/uv-binding.h

```diff
@@ -67,6 +67,12 @@ static inline uint64_t h2o_now_nanosec(h2o_loop_t *loop)
     return uv_now(loop) * 1000000;
 }
 
+static inline uint64_t h2o_evloop_get_execution_time_nanosec(h2o_loop_t *loop)
+{
+    /* unsupported at the moment */
+    return 0;
+}
+
```

Comment:
Do we need this to define the function here?

The function is specific to the "evloop" (see the name), therefore I think that it should not exist here.

Reply:
We don't. Sorry about that, this was a bad way to address the implicit declaration warning from the compiler. Does this make more sense? a987ac73